tests/test_snmpserver.rb

```diff
@@ -314,7 +314,7 @@ def test_snmpserver_global_enforce_priv_get_enabled
     if device_enabled
       refute_show_match(pattern: /no snmp-server globalEnforcePriv/)
     else
-      assert_show_match(pattern: /no snmp-server globalEnforcePriv/)
+      assert_show_match(pattern: /snmp-server globalEnforcePriv/)
```

Comment:
This doesn't look right - if it's enabled, we expect to NOT see 'no', but if it's disabled, we expect TO see 'yes'? Doesn't seem symmetric.